CHANGELOG.md

```diff
@@ -59,6 +59,7 @@ Released March 24, 2019
 - Add daylight saving time support for `CronSchedule` - [#729](https://github.com/PrefectHQ/prefect/pull/729)
 - Add `idempotency_key` and `context` arguments to `Client.create_flow_run` - [#757](https://github.com/PrefectHQ/prefect/issues/757)
 - Make `EmailTask` more secure by pulling credentials from secrets - [#706](https://github.com/PrefectHQ/prefect/issues/706)
+- Bump to latest version of `pytest` (4.3) - [#814](https://github.com/PrefectHQ/prefect/issues/814)
```

Comment:
Oops, I apologize @ColCarroll - I overlooked that this line should go in the "unreleased" section of the Changelog since we have released 0.5.0 already.